### PR TITLE
Flask/docker

### PR DESCRIPTION
**Pull request by NykoEugen**
Have problem with docker compose. My flask app doesn't start.
Django/docker had not create yet

### CODE REVIEW

**Comment by acman** [comment]
Don't see something suspicious. What errors did you face?

**Comment by NykoEugen** [comment]
It types "exited" in docker, and don't have any errors in log and in console. Before it was probably somthing like error with path, but now it isn't

**Comment by NykoEugen** [comment]
update, add django-docker. Have a problem too. But here has an error : "Error response from daemon: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "gun
icorn": executable file not found in $PATH: unknown"

**Comment by NykoEugen** [comment]
and maybe i had some like that error in flask-docker

File: django_project/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM python:latest
+
+WORKDIR /code
+
```
Comment (acman) [inline django_project/Dockerfile:4]: I think all required packages should be installed here

**Review by acman — APPROVED** [review]
Overall configs look good.

**Pull request merged**